app/flatpak-builtins-document-export.c

```diff
@@ -174,6 +174,7 @@ flatpak_builtin_document_export (int argc, char **argv,
   fd_list = g_unix_fd_list_new ();
   fd_id = g_unix_fd_list_append (fd_list, fd, error);
   close (fd);
+  fd = -1;
```

Comment:
I would prefer close(glnx_steal_fd (&fd))

Reply:
The docs for `glnx_close_fd()` say it is effectively `close(glnx_steal_fd (&fd))` but also with a check for `EBADF` so I'll go with that.